### DISCUSSION TITLE
100 plots and can't farm

### DISCUSSION THREAD

**Discussion by Teledyol** [Chia-Network/chia-blockchain#1976] [Q&A]
![Screen Shot 2021-04-16 at 7 33 45 PM](https://user-images.githubusercontent.com/3718370/115099460-dcddee80-9eea-11eb-9ad3-a8be79754236.png)

Haven't seen an acknowledgment in "Issues" so I thought I'd make it a discussion. Is it possible to revert to 1.0.3 to get this thing working again?

**Comment by stonehenge360** [comment]
what does your full node synchronization say?  

Is it still downloading the block chain?
